src/Features/VisualBasic/GenerateType/VisualBasicGenerateTypeService.vb

```diff
@@ -90,10 +90,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.GenerateType
                 document As SemanticDocument, simpleName As SimpleNameSyntax, cancellationToken As CancellationToken, ByRef generateTypeServiceStateOptions As GenerateTypeServiceStateOptions) As Boolean
             generateTypeServiceStateOptions = New GenerateTypeServiceStateOptions()
 
-            If simpleName.GetAncestorOrThis(Of ImportsStatementSyntax)() IsNot Nothing Then
```

Comment:
Interesting that we had explicitly blocked this scenario - do you know if there was some good reason for it?